Sources/HTTPTypes/HTTPFieldName.swift

```diff
@@ -498,6 +498,11 @@ extension HTTPField.Name {
     /// https://fetch.spec.whatwg.org/
     public static var xContentTypeOptions: Self { .init(rawName: "X-Content-Type-Options", canonicalName: "x-content-type-options") }
 
+    /// Proxy-Status
+    ///
+    /// https://www.rfc-editor.org/rfc/rfc9209.html
+    public static var proxyStatus: Self { .init(rawName: "Proxy-Status", canonicalName: "proxy-status") }
```

Comment:
Would you move this up to keep the list alphabetical?

Reply:
done!